test/OpenTelemetry.Exporter.Geneva.Tests/GenevaTraceExporterTests.cs

```diff
@@ -53,6 +53,34 @@ public void GenevaTraceExporter_constructor_Invalid_Input()
                     ConnectionString = null,
                 });
             });
+            Assert.Throws<ArgumentNullException>(() =>
```

Comment:
What's the purpose of this test?

Reply:
When i worked on the unit test, doing some test, for this i saw this is throwing the exception for invalid connect string as well, so leave it here. I have deleted this one, Thanks!